osidb/models.py

```diff
@@ -1753,6 +1767,25 @@ def create_flawcomment(flaw, external_system_id, comment, **extra_fields):
             flawcomment.meta_attr = comment
             return flawcomment
         except ObjectDoesNotExist:
+            # The "text" field is required for full validation, so save it if the caller forgets
+            # to explicitly copy it over from meta_attr.
+            extra_fields["text"] = extra_fields.get("text") or comment.get("text")
+
+            # The "order" field is required for full validation, but bugzilla calls it "count".
+            extra_fields["order"] = extra_fields.get("order") or comment.get("count")
+
+            # In-place upgrade of a temporary pending comment to preserve uuid.
+            pending = FlawComment.objects.pending().filter(
+                flaw=flaw, order=extra_fields["order"]
+            )
+            if pending.exists():
+                flawcomment = pending.first()
+                flawcomment.external_system_id = external_system_id
+                flawcomment.meta_attr = comment
```

Comment:
Is there any additional information in the comment JSON above `text`, `order`, `external_system_id`, timestamps (private) which is worth saving?

Reply:
I don't know. Is this a part of the ask in the ticket?

Reply:
We can live with some extra duplicate data. You can leave it as it is. Generally the tasks are quite vague and rarely describe technical details so it is up to the implementor.